op-node/rollup/derive/l1_block_info.go

```diff
@@ -400,7 +401,7 @@ func DepositsCompleteDeposit(seqNumber uint64, block eth.BlockInfo) (*types.Depo
 		// GasBenchMark_L1BlockIsthmus_DepositsComplete:test_depositsComplete_benchmark() (gas: 7768)
```

Comment:
Small nit: this comment is better to be next to the variable declaration for `DepositsCompleteGas`